miqa/settings.py

```diff
@@ -38,6 +38,9 @@ def before_binding(configuration: ComposedConfiguration) -> None:
         # oauth session
         configuration.OAUTH2_PROVIDER['ACCESS_TOKEN_EXPIRE_SECONDS'] = 1800  # 30 minutes
 
+        # drf
+        configuration.REST_FRAMEWORK['DEFAULT_PERMISSION_CLASSES'] = ['rest_framework.permissions.IsAuthenticated']
```

Comment:
Does this mean that you no longer need to specify `IsAuthenticated` on every view?

Reply:
Yup I will make this change